h2o-py/h2o/plot/_plot_result.py

```diff
@@ -0,0 +1,34 @@
+# -*- encoding: utf-8 -*-
+# mutable versions of py immutable types
+class _MObject(object): pass
+class _MTuple(tuple): pass
+class _MList(list): pass
+class _MDict(dict): pass
+class _MStr(str): pass
+class Error(EnvironmentError): pass
+
+def decorate_plot_result(res=None, figure=None):
+    def get_figure():
+        if figure != "RAISE_EXCEPTION_FLAG":
+            return figure
```

Comment:
magic string!
The pythonic way:
```python
RAISE_ON_FIGURE_ACCESS = object()

def get_figure():
     if figure is not RAISE_ON_FIGURE_ACCESS:
         return figure
     ...
     
 # client code
from h2o.plot import RAISE_ON_FIGURE_ACCESS
...
return decorate_plot_result(figure=RAISE_ON_FIGURE_ACCESS)
```
this is safe: no magic string, everything is verified by compiler

Reply:
52d6678